app/index.js

```diff
@@ -784,6 +784,20 @@ class GlobalBindings {
     this.requestMove = (user, channel) => {
       if (this.connected()) {
         user.model.setChannel(channel.model)
+
+        // get full channel path
+        let path = channel.name()
+        let parent = channel.parent()
+        while( parent.name && parent.name() != 'Root' ){
```

Comment:
If the user is already in the root channel, wouldn't `parent` be null? In that case we should probably skip adding the `channelName` alltogether.
Also, I don't think the name of the root channel is guaranteed to be `'Root'` (but its parent should always be null).

Reply:
### That's right, I get `TypeError: parent is undefined` if I go back to Root. Let's fix that.

Reply:
I fixed it.